react-native/react/constants/platform.native.desktop.js

```diff
@@ -20,26 +20,43 @@ const envedPathOSX = {
   prod: 'Keybase'
 }
 
+function buildWin32SocketRoot() {
+  let appdata = process.env.APPDATA
+  // Remove leading drive letter e.g. C:
+  if (/^[a-zA-Z]:/.test(appdata)) {
+    appdata = appdata.slice(2)
+  }
+  // Handle runModes, prod has no extension.
+  let extension = ''
+  if (runMode !== 'production') {
+    extension = runMode.charAt(0).toUpperCase() + runMode.substr(1)
+  }
+  let path = `\\\\.\\pipe\\kbservice${appdata}\\Keybase${extension}`
+  return path
+}
```

Comment:
If KEYBASE_SOCKET_FILE has bee set to something else in the environment, or by launching the service with option "socket-file", the names won't match. Shouldn't the same options be honored here?
https://github.com/keybase/client/blob/master/go/libkb/env.go#L433